jep/200/README.adoc

```diff
@@ -123,6 +123,18 @@ When a class is (first) rejected, a warning is printed to the system log.
 This is particularly useful when scanning large automated test results for regressions caused by this change.
 Typically the rejection will also trigger an exception which gets displayed in some other manner.
 
+== Extensibility
```

Comment:
This is not an official JEP section. It could be a subsection of `Specification` I suppose.

Reply:
Sections are just a skeleton. Adding new section is not a problem. Can indent it to the Specification, sure